.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: required
```

Comment:
Why is sudo required ?

Reply:
- [`mkDocs.sh` relies on docker](https://github.com/gisaia/ARLAS-web-components/blob/b536df1d473cf4400d175b9c148f35f9ead25d28/mkDocs.sh#L9)
- [`sudo: required` is necessary to use docker in travis-ci](https://docs.travis-ci.com/user/docker/)

Reply:
`sudo`-enabled machines allow to use `sudo`, but if you don't use it, the instructions won't be executed as root. See official documentation [here](https://docs.travis-ci.com/user/reference/overview/#Sudo-enabled).